graphics/blit.h

```diff
@@ -226,6 +226,8 @@ class BlendBlit {
 	static void blitAVX2(Args &args, const TSpriteBlendMode &blendMode, const AlphaType &alphaType);
```

Comment:
I feel like this might get a bit messy (especially since I'd rather keep the user-facing and internal headers separate), so I'd rather leave this alone for now.

Reply:
Fair enough, I trust your judgement on the matter.